src/snapred/ui/workflow/ReductionWorkflow.py

```diff
@@ -149,7 +162,7 @@ def _triggerReduction(self, workflowPresenter):
             )
 
             response = self.request(path="reduction/", payload=request_)
-            if response.code == ResponseCode.OK:
+            if isinstance(response.data, ReductionResponse):
```

Comment:
If I am understanding this correctly, you:
1. Attempt reduction
2. If it turns out you need an artificial normalization(As determined by the response type containing one workspace name) then you 
3. Create the base Artificial Normalization workspace for the next step
4. Allow the user to tweak the AN
5. Proceed with Reduction

I think it would be more straight forward if you didnt shoehorn the check into the same method as reduction.
1. User inputs values in first view -> method to check if there needs to be an AN, create one if necessary, update/skip AN view
  1. No AN Required?  The AN view is either skipped entirely or greyed out with a message letting the user know no AN was required.
  2. Yes AN Required? The AN is initialized and the user tweaks it in the AN view
2. The AN Tweak View is utilized or skipped -> Proceed with Reduction
3. Save.

This way you only ever call reduction in one spot and you have a linear non branching chain of events.  It should reduce the complexity of the code.

Reply:
According to the acceptance criteria of the parent story
> The user can select "artificial normalisation" via a UI prompt and submit the two parameters above.
Unfortunately Im not sure it was supposed to be a step in reduction 😔, please check with Malcolm to see if this is acceptable or if you need to move it.